custom.ini

```diff
@@ -1,2 +1,4 @@
 # Increase upload_max_filesize to install the LearnDash plugin
-upload_max_filesize=512M
+upload_max_filesize=128M
+post_max_size=128M
+max_execution_time=300
```

Comment:
Increase `max_execution_time` as well to process larger file size:
https://www.php.net/manual/en/info.configuration.php#ini.max-execution-time